pulsar/internal/pulsar_proto/PulsarApi.pb.go

```diff
@@ -18,8 +18,8 @@
 
 // Code generated by protoc-gen-go. DO NOT EDIT.
 // versions:
-// 	protoc-gen-go v1.28.1
-// 	protoc        v3.21.9
+// 	protoc-gen-go v1.31.0
+// 	protoc        v4.24.3
```

Comment:
Could you use the same `protoc` to generate the `PulsarApi.pb.go`? Not sure if the protobuf introduces any breaking change between 3 and 4.

We'd better update the instruction here

https://github.com/apache/pulsar-client-go/blob/b4d45cd360599b52e3f160db9d39d619889c556a/pulsar/internal/pulsar_proto/README.md?plain=1#L4

to

```bash
go install google.golang.org/protobuf/cmd/protoc-gen-go@v1.28.1
```

And to keep using `protoc` 3.21, here is an example on macOS:

```bash
brew install protobuf@21
/opt/homebrew/opt/protobuf@21/bin/protoc --go_out=. --go_opt=paths=source_relative PulsarApi.proto
```

Reply:
Done! Note that it really only changed the version comments, everything else stayed the same: https://github.com/apache/pulsar-client-go/pull/1208/commits/880d732ed54d774b539d09897f273e24e5b7608f.

Reply:
Yeah, when I regenerated it locally, I also found only the version comments were changed.